packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -941,6 +942,16 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
         e.stopPropagation()
         return
       default:
+        const keyString = String.fromCharCode(keyCode)
+        if (/[a-zA-Z]/.test(keyString)) {
```

Comment:
what about numbers or any other printable characters? they can be part of the dropdown items as well.

Reply:
I will try to find another regex.